packages/db/src/models/reconciliation_report.ts

```diff
@@ -0,0 +1,37 @@
+import { Knex } from 'knex';
+import { BasePgModel } from './base';
+import { TableNames } from '../tables';
+
+import {
+  PostgresReconciliationReport,
+  PostgresReconciliationReportRecord,
+} from '../types/reconciliation_report';
+
+// eslint-disable-next-line max-len
+class ReconciliationReportPgModel extends BasePgModel<PostgresReconciliationReport, PostgresReconciliationReportRecord> {
+  constructor() {
+    super({
+      tableName: TableNames.reconciliationReports,
+    });
+  }
+
+  create(
+    knexOrTransaction: Knex | Knex.Transaction,
+    item: PostgresReconciliationReport
+  ) {
+    return super.create(knexOrTransaction, item, '*');
+  }
+
+  upsert(
+    knexOrTransaction: Knex | Knex.Transaction,
+    reconciliationReport: PostgresReconciliationReport
+  ) {
+    return knexOrTransaction(this.tableName)
+      .insert(reconciliationReport)
+      .onConflict('name')
+      .merge()
+      .returning('*');
+  }
+}
+
```

Comment:
Don't we need a `delete`?

Reply:
On suggestion from Jonathan, I replicated what was in the rules model. TBH, I'm not sure if the create is needed, I kind of think the inherited method would just work, but again I copied it from rules. I _think_ the delete should just work as inherited, and I didn't see any delete functions being declared in the other models.

Reply:
Hmm yeah, that's true. I was looking at models/granule.ts where we re-implement a `delete`. But that doesn't do much except enforce the unique columns. 

It's probably fine to use the base model's delete? What if you were to pass a recon report object that didn't include `name` (the unique column)? It seems like it would then delete the first record that matched, which could be incorrect. 

I'm reaching so that may not be a concern. Using other record types as an example is probably fine.